Nmbrs/tf-modules
# maint (resource_group): added caf naming convention provider

# Description

Added azure caf naming convention provider to support the creation of the resource group name to follow the CAF naming convention.

# Motivation and Context


Be compliant with the CAF naming convention without zero costs.


# How Has This Been Tested?



- push the branch to your local
- make sure that the source is pointing to your local path where the azure resource group module exists
- copy/paste the code into main.tf
```hcl
module "rg" {
  #source   = "git::github.com/Nmbrs/tf-modules//azure/resource-group"
  source      = "../tf-modules/azure/resource-group"
  name        = "Heimdall"
  country     = "NL"
  environment = "Dev"
  squad       = "Infra"
  product     = "Microservice Platform"
  extra_tags = {
    Datadog = "Monitored"
  }
}
```
- `terraform init -upgrade && terraform apply -auto-approve

# Types of changes


# Checklist:
